apis/Google.Cloud.Vision.V1/Google.Cloud.Vision.V1.Tests/ImageAnnotatorClientTest.cs

```diff
@@ -174,20 +174,13 @@ public class ImageAnnotatorClientTest
             AssertAnnotationsSuccess(allAnnotations, await asyncCall(s_allAnnotationsImage, null, 0, null));
             AssertAnnotationsSuccess(allAnnotations.Take(1), syncCall(s_allAnnotationsImage, null, 1, null));
             AssertAnnotationsSuccess(allAnnotations.Take(1), await asyncCall(s_allAnnotationsImage, null, 1, null));
-            AssertAnnotationError(syncCall(s_errorImage, null, 0, null));
-            AssertAnnotationError(await asyncCall(s_errorImage, null, 0, null));
+            Assert.Throws<AnnotateImageException>(() => syncCall(s_errorImage, null, 0, null));
```

Comment:
No, it's fine. We *could* check that the error is the one we expected, but most of that is checked elsewhere.